test/slow/nvm use/Running 'nvm use --save --lts' should work as expected

```diff
@@ -0,0 +1,82 @@
+#!/bin/sh
+
+set -ex
+
+clean () {
+  if [ "$(basename "$(pwd)")" = "tmp" ]; then
+    cd ..
+  fi
+  rm -rf .nvmrc tmp;
+}
+die () { echo "$@" ; clean; exit 1; }
+
+\. ../../../nvm.sh
+
+nvm deactivate >/dev/null 2>&1 || die 'deactivate failed'
+
+##
+
+test_version () {
+  clean
+  VERSION=${1-}
+  nvm install --save "$VERSION" || die "\`nvm install --save $VERSION\` failed"
+  OUTPUT="$(cat .nvmrc)"
+  EXPECTED_OUTPUT=$(nvm_ls_current)
+
+  [ "_$OUTPUT" = "_$EXPECTED_OUTPUT" ] \
+    || die "\`nvm use use --save $VERSION\`+ \`cat .nvmrc\` did not output '$EXPECTED_OUTPUT'; got '$OUTPUT'"
+}
+
+test_version '--lts'
+test_version 'lts/argon'
+test_version 'lts/*'
+test_version 'node'
+test_version 'iojs'
+
+##
+clean
+##
+
+nvm install --save --lts
+mkdir tmp
+cd tmp || die 'Unable to access tmp directory';
+test_version
+
+##
+clean
+##
+
+nvm alias test node
+nvm use test --save || die "\`nvm use test --save\` failed"
+OUTPUT="$(cat .nvmrc)"
+EXPECTED_OUTPUT=$(nvm_ls_current)
+[ "_$OUTPUT" = "_$EXPECTED_OUTPUT" ] \
+  || die "\`nvm use use --save $VERSION\` + \`cat .nvmrc\` did not output '$EXPECTED_OUTPUT'; got '$OUTPUT'"
+nvm unalias test
```

Comment:
might be nice to put each of these tests in their own file - ie, so `clean` is only needed once per file, at the end